package.json

```diff
@@ -3,19 +3,33 @@
   "private": true,
   "author": "Nick Barry",
   "license": "MIT",
+  "scripts": {
+    "compile": "hardhat compile && yarn copyAbi && yarn typechain",
+    "copyAbi": "ts-node copyAbi.ts",
+    "typechain": "typechain --target ethers-v5 --out-dir typechain \"./abiTypechain/[!package]*.json\""
+  },
   "devDependencies": {
     "@nomiclabs/hardhat-ethers": "^2.0.2",
     "@nomiclabs/hardhat-waffle": "^2.0.1",
     "@solidstate/spec": "link:spec",
+    "@typechain/ethers-v5": "^7.0.0",
+    "@types/chai": "^4.2.18",
+    "@types/mocha": "^8.2.2",
+    "@types/node": "^15.6.0",
+    "@types/rimraf": "^3.0.0",
     "chai": "^4.3.4",
-    "eslint": "^7.24.0",
```

Comment:
Why remove eslint?

Reply:
I feel like eslint is redundant when using typescript, as typescript compiler already detects code errors (+ type errors which eslint doesnt check) and prettier takes care of the code formatting.

Reply:
I think you'd be right, if prettier were actually capable of making things prettier.  For now the removal of eslint is fine, as it reduces complexity, but I might add it again at some point.